VoteStakeRegistry/sdk/accounts.tsx

```diff
@@ -76,6 +78,28 @@ export const getRegistrarPDA = async (
   }
 }
 
+export const getMaxVoterWeightPDA = async (
```

Comment:
This PDA is helium specific right? if so I think it would be good to have a docstring stating this. If it is not then ignore this comment.